packages/frontend/src/types/index.ts

```diff
@@ -1,41 +1,38 @@
+export interface IUser {
+  id: string;
+  name: string;
+  // telegramId: string;
```

Comment:
I wouldn't remove this field because it can be useful, maybe mark it as not necessary instead?  And we can fill it when user login to the app for the first time. In that case if user change username we can still recognize him, since his telegramId will be the same

``` telegramId?: string;```